Collection.h

```diff
@@ -0,0 +1,24 @@
+//
+//  Collection.h
+//  Trippy
+//
+//  Created by Catherine Lu on 7/7/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "Location.h"
+#import "Parse/Parse.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Collection : NSObject
+@property (strong, nonatomic) NSArray *locations;
+@property (strong, nonatomic) NSString *userId;
+@property (strong, nonatomic) NSString *objectId;
```

Comment:
Similar comment here... what is `objectId` supposed to be? Maybe make it more specific to its function.